pycbc/distributions/masses.py

```diff
@@ -75,6 +75,10 @@ def __init__(self, mass1=None, mass2=None):
             raise ValueError("must provide limits for mass2")
         super(UniformComponentMasses, self).__init__(mass1=mass2, mass2=mass2)
 
+    def cdfinv(self, param, value):
+        raise NotImplementedError("This distribution has not been "
+                                  "implemented for multinest.")
```

Comment:
Change the message here to: `An inverse CDF function has not been implemented for this distribution.` This isn't multinest specific.